examples/testdata/path_matcher/openapi_swagger.json

```diff
@@ -5,7 +5,7 @@
     "title": "Example - Path Matcher - Bookstore",
     "version": "1.0.0"
   },
-  "host": "path-matcher.endpoints.apiproxy-231719.cloud.goog",
+  "host": "path-matcher.endpoints.nareddyt-cloud-esf.cloud.goog",
```

Comment:
can we not to use personal userid here?

Reply:
I don't have access to deploy to `apiproxy-number` project. And find replace is too hard, multiple substrings and different case mismatch.

How about this: I'll make another PR to deploy all these examples to `cloudesf-testing`. Once that's merged to master, i'll update the diff.

Reply:
ok

Reply:
I'll update diff once #225 is merged. Ignore until then.

Reply:
Done